resharper/src/resharper-unity/ShaderLab/Psi/DeclaredElements/ShaderLabDeclaredElementType.cs

```diff
@@ -0,0 +1,27 @@
+﻿using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.Resources;
+using JetBrains.UI.Icons;
+
+namespace JetBrains.ReSharper.Plugins.Unity.ShaderLab.Psi.DeclaredElements
+{
+    public class ShaderLabDeclaredElementType : DeclaredElementType
+    {
+        // TODO: Proper icon
```

Comment:
Do we want some unity-specific icon here?

Reply:
Yep. I'll add a request